src/org/zaproxy/zap/extension/jxbrowser/ZapBrowserPanel.java

```diff
@@ -87,6 +89,15 @@ public Browser getBrowser() {
                 browser = new Browser();
             }
         }
+        if (LOGGER.isDebugEnabled()) {
```

Comment:
This check can be moved to the `onMessage` method instead, worth adding the listener even if debug level is not enabled as the level can be changed any time.